libs/chrome-ext/src/lib/options/options.html

```diff
@@ -37,12 +38,17 @@
     </fieldset>
     <fieldset>
       <legend>YouTube</legend>
+      <p>
+        Optional. When a <a href="https://developers.google.com/youtube/v3/getting-started">YouTube API key</a> is
+        defined, you can index all videos from a YouTube channel or YouTube.
```

Comment:
`from a YouTube channel or YouTube.` didn't you forget a word at the end? like `YouTube channel or YouTube play list.`?